dist/rpm/openQA.spec

```diff
@@ -382,9 +382,6 @@ export LANG=en_US.UTF-8
 rm -rf %{buildroot}/%{_sysusersdir}
 %endif
 
-mkdir -p %{buildroot}%{_datadir}/openqa%{_sysconfdir}/openqa
-ln -s %{_sysconfdir}/openqa/openqa.ini %{buildroot}%{_datadir}/openqa%{_sysconfdir}/openqa/openqa.ini
-ln -s %{_sysconfdir}/openqa/database.ini %{buildroot}%{_datadir}/openqa%{_sysconfdir}/openqa/database.ini
```

Comment:
Please test based on RPM packages from https://download.opensuse.org/repositories/devel:/openQA:/GitHub:/os-autoinst:/openQA:/PR-6328/openSUSE_Tumbleweed/devel:openQA:GitHub:os-autoinst:openQA:PR-6328.repo regarding still being able to start the webUI, gru, database connection, worker, client, etc.

Reply:
I have already done that so I'm removing the "not-ready" label now. The packages are installable and used the correct config files (including the database and client). The worker also still works (although it is only affected for the client config anyway).